draft-ietf-taps-impl.md

```diff
@@ -489,7 +489,7 @@ Instead, calling the ConnectionReceived event may be delayed until the first Mes
 
 ## Handling racing with "unconnected" protocols {#unconnected-racing}
 
-While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as a local route to the peer endpoint is confirmed.
+While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as connectivity with the peer endpoint is established.
 
 However, if a peer is not reachable over the network using the unconnected protocol, or data cannot be exchanged for any other reason, the application may want to attempt using another candidate Protocol Stack. The implementation should maintain the list of other candidate Protocol Stacks that were eligible to use.
```

Comment:
I do not think this update helped. In fact, I think it made it worse. What does "connectivity with the peer endpoint is established" mean for an unconnected protocol?

Reply:
Agreed. How about "as soon as a route on which to send and receive packets is present, and any necessary state like local port reservations has been established."